packages/sfp-cli/src/core/package/promote/PromoteUnlockedPackageImpl.ts

```diff
@@ -1,13 +1,14 @@
-import SFPLogger from '@flxbl-io/sfp-logger';
 import { SfProject } from '@salesforce/core';
 import { PackageSaveResult, PackageVersion } from '@salesforce/packaging';
 import SFPOrg from '../../org/SFPOrg';
+import SFPLogger, {COLOR_HEADER} from "@flxbl-io/sfp-logger";
 
 export default class PromoteUnlockedPackageImpl {
     public constructor(
         private project_directory: string,
         private package_version_id: string,
-        private devhub_alias: string
+        private devhub_alias: string,
+        private fail_if_is_alreadyPromoted: boolean
```

Comment:
Why variable name in the constructor have this different naming convention?
Can I use always camel case?
@azlam-abdulsalam